website/docs/r/compute_subnetwork.html.markdown

```diff
@@ -1,98 +1,177 @@
+  }
+
+  resource "google_compute_subnetwork" "network-with-private-secondary-ip-ranges" {
+    name = "%s"
+    ip_cidr_range = "10.2.0.0/16"
+    region = "us-central1"
+    network = "${google_compute_network.custom-test.self_link}"
+    secondary_ip_range {
+      range_name = "tf-test-secondary-range-update1"
+      ip_cidr_range = "192.168.10.0/24"
+    }
+  }
+```
 
-* `region` - (Optional) The region this subnetwork will be created in. If
-    unspecified, this defaults to the region configured in the provider.
+## Argument Reference
 
-* `private_ip_google_access` - (Optional) Whether the VMs in this subnet
-    can access Google services without assigned external IP
-    addresses.
+The following arguments are supported:
 
-* `enable_flow_logs` - (Optional, [Beta](/docs/providers/google/index.html#beta-features))
-    Set to `true` to enable [flow logs](https://cloud.google.com/vpc/docs/using-flow-logs)
-    for this subnetwork.
+* `ip_cidr_range` -
+  (Required)
+  The range of internal addresses that are owned by this subnetwork.
+  Provide this property when you create the subnetwork. For example,
+  10.0.0.0/8 or 192.168.0.0/16. Ranges must be unique and
+  non-overlapping within a network. Only IPv4 is supported.
+* `name` -
+  (Required)
+  The name of the resource, provided by the client when initially
+  creating the resource. The name must be 1-63 characters long, and
+  comply with RFC1035. Specifically, the name must be 1-63 characters
+  long and match the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which
+  means the first character must be a lowercase letter, and all
+  following characters must be a dash, lowercase letter, or digit,
+  except the last character, which cannot be a dash.
+* `network` -
+  (Required)
+  The network this subnet belongs to.
+  Only networks that are in the distributed mode can have subnetworks.
 
 
 - - -
 
-* `secondary_ip_range` - (Optional, [Beta](/docs/providers/google/index.html#beta-features)) An array of configurations for secondary IP ranges for VM instances contained in this subnetwork. Structure is documented below.
+* `description` -
+  (Optional)
+  An optional description of this resource. Provide this property when
+  you create the resource. This field can be set only at resource
+  creation time.
+* `enable_flow_logs` -
+  (Optional)
+  Whether to enable flow logging for this subnetwork.
+* `secondary_ip_range` -
+  (Optional)
+  An array of configurations for secondary IP ranges for VM instances
+  contained in this subnetwork. The primary IP of such VM must belong
+  to the primary ipCidrRange of the subnetwork. The alias IPs may belong
+  to either primary or secondary ranges.  Structure is documented below.
+* `private_ip_google_access` -
+  (Optional)
+  Whether the VMs in this subnet can access Google services without
+  assigned external IP addresses.
+* `region` -
+  (Optional)
+  URL of the region where the regional address resides.
```

Comment:
This looks like a copy/paste error

Reply:
Haha, I think that was in magic modules already, but it was _someone's_ copy/paste error.